CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.21.0](https://github.com/sonata-project/SonataBlockBundle/compare/3.20.0...3.21.0) - 2020-08-25
+### Added
+- [[#732](https://github.com/sonata-project/SonataBlockBundle/pull/732)] Added support for "twig/twig:^3.0" ([@phansys](https://github.com/phansys))
+- [[#730](https://github.com/sonata-project/SonataBlockBundle/pull/730)] Added support for symfony/options-resolver:^5.1 ([@phansys](https://github.com/phansys))
+
+### Fixed
+- [[#717](https://github.com/sonata-project/SonataBlockBundle/pull/717)] Missing `?` in type hints where null is default value ([@wbloszyk](https://github.com/wbloszyk))
```

Comment:
This is pedantic, no ?

Reply:
IMHO it is not pedantic becouse with `?` people can call methods with full available param even if param is null what was unavailable before #717 PR

Reply:
I'm not sure to understand. The `?` is not needed when `null` is the default value.
```
function foo(string $f = null) {}

foo();
foo(null);
```
Both work.

Reply:
Agree, I think about other PR where default value is not null. In this PR all change have `= null` so it is pedantic.